base/src/main/java/com/smartdevicelink/proxy/rpc/WindowState.java

```diff
@@ -0,0 +1,49 @@
+package com.smartdevicelink.proxy.rpc;
+
+import android.support.annotation.NonNull;
+
+import com.smartdevicelink.proxy.RPCStruct;
+
+import java.util.Hashtable;
+
```

Comment:
```suggestion
/**
 *
 * <p><b>Parameter List</b></p>
 *
 * <table border="1" rules="all">
 *  <tr>
 *      <th>Param Name</th>
 *      <th>Type</th>
 *      <th>Description</th>
 *      <th>Required</th>
 *      <th>Version Available</th>
 *  </tr>
 *  <tr>
 *      <td>approximatePosition</td>
 *      <td>Integer</td>
 *      <td>The approximate percentage that the window is open - 0 being fully closed, 100 being fullyopen</td>
 *      <td>Y</td>
 *      <td></td>
 *  </tr>
 *  <tr>
 *      <td>deviation</td>
 *      <td>Integer</td>
 *      <td>The percentage deviation of the approximatePosition. e.g. If the approximatePosition is 50and the deviation is 10, then the window's location is somewhere between 40 and 60.</td>
 *      <td>Y</td>
 *      <td></td>
 *  </tr>
 * </table>
 * @since SmartDeviceLink 7.0.0
 */
```
Please use the [RPC generator](https://github.com/smartdevicelink/sdl_java_suite/tree/master/generator) in the sdl_java_suite project to ensure the RPC matched the spec exactly. The generator will also update the rest of the JavaDocs in this file to ensure everything matches the RPC Spec exactly